### PR TITLE
Mobile sadface

### PR DESCRIPTION
**Pull request by bfaulk04**

### CODE REVIEW

File: src/styles/site.css [hunk 1]
```diff
@@ -46,11 +46,18 @@
...
+=======
 .page-who-we-are .testimonial-section a,
 .page-who-we-are .testimonial-section a strong {
   @apply text-white;
 }
 
 .page-thrift-store h3 {
   @apply text-gray-900 text-2xl lg:text-3xl font-extrabold leading-9;
+>>>>>>> main
 }
```
Comment (mdxprograms) [inline src/styles/site.css:63]: @bfaulk04 just need to fix up conflicts then we are gtg

**Review by mdxprograms — CHANGES_REQUESTED** [review]
just need to fix conflicts then we are gtg

File: src/styles/site.css (hunk 1 above)
Comment (bfaulk04) [reply src/styles/site.css:63]: idk how to do that, all looks as intended to me

Comment (mdxprograms) [reply src/styles/site.css:63]: just remove the portions that aren't needed and re-commit. That would be the `<<<` line portions or `>>>>>` . The `HEAD`  portion shows what is on your branch. The `main` is what's on main. So if there are styles that you've changed that are conflicting with what's there you can remove the previous ones and keep yours or keep both if they don't conflict in general but remove the conflict guards and then re-commit.

**Pull request closed without merging**